### ISSUE TITLE
Improve readme.md

### ISSUE THREAD

**Issue by osaid-r** [stellargo/meseek#35]
Is this a request for change in some structure (yay/nay) :
Yes

Description:
Readme isn't clear ! No idea what the project is about.

**Comment by stellargo** [comment]
Yeah its a work in progress, I have yet to do a lot of tasks before coming to improving documentation. The GIF is a proof of concept (which is the whole code at this time), and I am gradually working on it :)

**Comment by stellargo** [comment]
Duplicate of https://github.com/stellargo/meseek/issues/17

**Comment by osaid-r** [comment]
I think writing the Readme properly should be top priority if any contributions from others are expected.
